pom.xml

```diff
@@ -59,15 +59,6 @@
             </roles>
             <timezone>-6</timezone>
         </developer>
-        <developer>
-            <id>jlowe</id>
-            <name>Jason Lowe</name>
-            <email>jlowe@nvidia.com</email>
-            <roles>
-                <role>Committer</role>
```

Comment:
Per my understanding of https://maven.apache.org/xsd/maven-4.0.0.xsd the text within `<role>` can be chosen arbitrarily. So instead of dropping the entry entirely we could say Emeritus Committer

Reply:
it doesn't matter to me to much, its really more of a project process type thing.  Are there any benefits to it over removing?

Reply:
I find it more graceful to continue giving credit for significant contributions and many projects seem to have an Emeritus status.

Reply:
Everything for contributions is in git history.  I don't see putting him as emeritus in the pom file for list of committers as being a big deal personally.  There are no plans for him to come back.

Reply:
I'm going to merge this, if someone has a strong feeling we can revert and put him back in as Emeritus